src/android/aidl/types/arrays.md

```diff
@@ -0,0 +1,19 @@
+# Array Types
+
+The array types (`T[]`, `byte[]`, and `List<T>`) get translated to the
+appropriate Rust array type depending on how they are used in the function
+signature:
+
+| Position               | Rust Type     |
+| ---------------------- | ------------- |
+| `in` argument          | `&[T]`        |
+| `out`/`inout` argument | `&mut Vec<T>` |
+| Return                 | `Vec<T>`      |
```

Comment:
Also parcelable fields.

Reply:
What about parcelable fields? I'm not sure how that's relevant on this slide.

Reply:
I'm saying if you have a parcelable field that's an array, it will also use a `Vec<T>`. E.g.
```
parcelable Foo {
    int a[];
}
```

Reply:
Ah, that makes sense. I think I'll keep the slide specific to arrays in function signatures, but I'll add this to the speaker notes.